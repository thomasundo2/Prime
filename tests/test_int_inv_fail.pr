int main(){
    int test1;
    int test2;
    test1 = 2;
    test2 = 7;
    print(test1 ` test2);
    return 0;
}
