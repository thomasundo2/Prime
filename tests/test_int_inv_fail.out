Fatal error: exception Failure("illegal binary operator int ` int in test1 ` test2")
